pkg/profiler/cpu/cpu.go

```diff
@@ -304,6 +308,13 @@ func (p *CPU) addUnwindTableForProcess(pid int) {
 			level.Debug(p.logger).Log("msg", "failed to add unwind table due to a procfs race", "pid", pid, "err", err)
 		} else if errors.Is(err, errTooManyExecutableMappings) {
 			level.Warn(p.logger).Log("msg", "failed to add unwind table due to having too many executable mappings", "pid", pid, "err", err)
+		} else if errors.Is(err, ErrTextSectionNotFound) {
+			if p.processErrorTracker[err]%50 == 0 {
+				level.Error(p.logger).Log("msg", "failed to add unwind table due to unavailable .text section", "pid", pid, "err", err)
```

Comment:
nit: It would be nice to see the number of encounters so far in the logs.
```suggestion
				level.Error(p.logger).Log("msg", "failed to add unwind table due to unavailable .text section", "pid", pid, "err", err, "encounters", p.processErrorTracker[err])
```

Reply:
I like this